libsofia-sip-ua/sresolv/sres.c

```diff
@@ -2297,7 +2296,7 @@ int sres_parse_config(sres_config_t *c, FILE *f)
   c->c_port = _sres_default_port;
 
   if (f != NULL) {
-    for (line = 1; fgets(buf, sizeof(buf), f); line++) {
+    for (; fgets(buf, sizeof(buf), f);) {
```

Comment:
Became a `while` loop?

Reply:
changed to `while` loop for better readability